fe/fe-core/src/main/java/com/starrocks/catalog/FunctionSet.java

```diff
@@ -450,7 +450,9 @@ public class FunctionSet {
             .put(Type.CHAR, Type.ARRAY_VARCHAR)
             .put(Type.DATE, Type.ARRAY_DATE)
             .put(Type.DATETIME, Type.ARRAY_DATETIME)
-            .put(Type.DECIMAL32, Type.ARRAY_DECIMALV2)
+            .put(Type.DECIMAL32, Type.ARRAY_DECIMAL32)
```

Comment:
ARRAY_DECIMALV2 is deprecated? Is there a compatibility issue here ?

Reply:
emmm, may I add item (Type.DECIMALV2 -> Type.ARRAY_DECIMALV2) ?